lsp-haskell.el

```diff
@@ -238,7 +310,23 @@ and `lsp-haskell-server-args' and `lsp-haskell-server-wrapper-function'."
                                 ("haskell.liquidOn" lsp-haskell-liquid-on t)
                                 ("haskell.diagnosticsOnChange" lsp-haskell-diagnostics-on-change t)
                                 ("haskell.maxNumberOfProblems" lsp-haskell-max-number-of-problems)
-                                ("haskell.hlintOn" lsp-haskell-hlint-on t)))
+                                ("haskell.hlintOn" lsp-haskell-hlint-on t)
```

Comment:
Should we change this to `haskell.plugin.hlint.globalOn`?

Reply:
If so let's move the hlint `defcustom` under `lsp-haskell-plugins` too.

Reply:
This is one of the legacy config items, which will have to be moved/deprecated some time. See comments at https://github.com/haskell/haskell-language-server/pull/691#pullrequestreview-556488463